### PR TITLE
Fix streaming related Ballerina samples

### PR DESCRIPTION
**Pull request by mohanvive**
To align with best practices.

### CODE REVIEW

**Review by yasassri — APPROVED** [review]

File: examples/filter_streams_on_conditions/filter_streams_on_conditions.bal [hunk 1]
```diff
@@ -64,7 +63,7 @@ function main (string... args) {
 }
 
 //Print the output events.
-function printStatusCount (StatusCount s) {
-    io:println("Event received; status: " + s.status +" and total occurrences: "+s.totalCount);
+function printStatusCount(StatusCount s) {
+    io:println("Event received; status : " + s.status + " and total occurrences : " + s.totalCount);
```
Comment (lafernando) [inline examples/filter_streams_on_conditions/filter_streams_on_conditions.bal]: It should be "status: ", there shouldn't be an extra space there. Please fix this and similar places.

File: examples/join_multiple_streams/join_multiple_streams.bal [hunk 2]
```diff
@@ -40,8 +40,8 @@ function initRealtimeProductionAlert() {
         group by r.name
         having ((totalRawMaterial - totalConsumption) * 100.0 / totalRawMaterial) > 5
         => (MaterialUsage[] materialUsages) {
-            //The 'materialUsages' is the output that matches the defined streaming rules. It is published to `materialUsageStream` stream.
-            //The selected clause should match the structure of the `MaterialUsage` type.
+        //The 'materialUsages' is the output that matches the defined streaming rules. It is published to `materialUsageStream` stream.
```
Comment (lafernando) [inline examples/join_multiple_streams/join_multiple_streams.bal]: There should be a space after "//". Please fix in all the places.

File: examples/join_multiple_streams/tests/join_multiple_streams_test.bal [hunk 3]
```diff
@@ -5,7 +5,7 @@ import ballerina/http;
 boolean serviceStarted;
 
 function startService(){
```
Comment (lafernando) [inline examples/join_multiple_streams/tests/join_multiple_streams_test.bal]: Fix code formatting, fix other similar places too.

Comment (lafernando) [inline examples/join_multiple_streams/tests/join_multiple_streams_test.bal]: Fix code formatting, fix other similar places too.

**Pull request merged**